tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+
+			$this->opts_prefix . 'content_after' => array(
+				'type'				=> 'text',
+				'label'				=> esc_html__( '<span> text to display After/Below iframe', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default there is no content after the iframe. It will be output as a <span> with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
+				'validation_type'	=> 'textarea', // runs wp_kses()
+			),
+
+			// see https://central.tri.be/issues/62402
+			$this->opts_prefix . 'iframe_px' => array(
+				'type'				=> 'text',
+				'label'				=> esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
+				'default'			=> 200, // required else positive_int validation will throw an error each time settings are saved if this field is left blank
+				'tooltip'			=> esc_html__( 'The default is 200.', 'tribe-extension' ),
+				'validation_type'	=> 'positive_int',
+			),
+
+			$this->opts_prefix . 'locale' => array(
+				'type'				=> 'dropdown',
+				'label'				=> esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
+				'default'			=> 'en_US',
+				'tooltip'			=> sprintf( esc_html__( 'This affects the API call. The default is %sUnited States (en_US)%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
+				'validation_type'	=> 'options',
+				'options'			=> $this->eb_locale_country_name_array(),
+			),
+
+			$this->opts_prefix . 'ticket_single_location' => array(
+				'type'				=> 'dropdown',
+				'label'				=> esc_html__( 'Single Event Display Location', 'tribe-extension' ),
+				'default'			=> 'tribe_events_single_event_after_the_meta',
+				'tooltip'			=> sprintf( esc_html__( 'Where should the Eventbrite Tickets iframe display on your Single Events pages? The default is %sAfter the Meta%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
+				'validation_type'	=> 'options',
+				'options'			=> $this->single_event_display_locations_array(),
+			),
+
+			$this->opts_prefix . 'show_admin_cost_field' => array(
+				'type'				=> 'checkbox_bool',
+				'label'				=> esc_html__( 'Display The Events Calendar\'s "Event Cost" field.', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default, The Events Calendar\'s "Event Cost" field is hidden when Eventbrite Tickets or Event Tickets Plus is active. Check this box to re-enable it.', 'tribe-extension' ),
+				'validation_type'	=> 'boolean',
+			),
+
+			$this->opts_prefix . 'show_tickets_private_events' => array(
+				'type'				=> 'checkbox_bool',
+				'label'				=> esc_html__( 'Display Tickets for Private Events', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default, Eventbrite Tickets only displays tickets for events marked Public on Eventbrite. Check this box to also display tickets for the Private events that you have imported.', 'tribe-extension' ),
+				'validation_type'	=> 'boolean',
+			),
+
+		);
+
+		$setting_helper->add_fields(
+			$fields,
+			'imports',
+			'tribe_aggregator_disable_header', // if Event Aggregator is enabled, below Meetup, else below CSV
```

Comment:
I think this common is one that needs to be removed, as it seems to no longer be relevant.